src/container/EdgeRenderer/index.tsx

```diff
@@ -75,6 +76,15 @@ const EdgeRenderer = (props: EdgeRendererProps) => {
   const { connectionLineType, defaultMarkerColor, connectionLineStyle, connectionLineComponent } = props;
   const renderConnectionLine = connectionNodeId && connectionHandleType;
 
+  let { connectionLineContainerStyle } = props
```

Comment:
I think we can simplify this and change the default behaviour a bit. The connection line should always be above the other elements. I would set the z-index via CSS to 1001 (so that it's always above all nodes) and just pass the container style to the svg container. If you need a higher z-index, you can use the `connectionLineContainerStyle`.

Reply:
Let's see if I understood this correctly.

You want me to remove the zIndex default logic in line 80-87 and replace with a simple CSS rule (in a CSS file) that sets the zIndex to 1001, right?